### ISSUE TITLE
Cuestick-cueball Collision Equation

### ISSUE THREAD

**Issue by derek-mcblane** [ekiefl/pooltool#162]
I think the comment [here](https://github.com/ekiefl/pooltool/blob/8d973a0450ed03a64505f2fdbc9828bec5483227/pooltool/physics/resolve/stick_ball/instantaneous_point/__init__.py#L82-L84) is wrong, and that Leckie and Greenspan is right. See [TP A-30](https://billiards.colostate.edu/technical_proofs/new/TP_A-30.pdf) equation 7 and notice that the mass has cancelled, just like it would in Leckie and Greenspan if F is substituted into the velocity equation.  It's also worth mentioning that Leckie and Greenspan confusingly use F for both force as a function of time, and as the impulse resulting from integrating that force over the duration of the collision.

I tried adjusting the equation and the ball does seem to have a low amount of speed, but I think this might be a cue speed issue from the stroke UI.  I adjusted the swing speed factor and got decent results. I think the forward swing needs to be more sensitive than the backswing for it to look and feel right. Edit: maybe something like mouse acceleration would be appropriate.

**Comment by derek-mcblane** [comment]
Update: I think there is a _different_ issue with the Leckie and Greenspan. I'm not too confident in my result, but I converted from the TP A-30 equation to the Leckie and Greenspan equation, and got slightly a different result; for the 2*b*c*sin(theta)*cos(theta) term, I got that it should be added, not subtracted.

Initial testing with both the mass change and this change applied gives results that feel right.
